lib/Bindings/Python/pyproject.toml

```diff
@@ -9,3 +10,10 @@ requires = [
     "PyYAML",
 ]
 build-backend = "setuptools.build_meta"
+
+# Enable version inference from Git.
+[tool.setuptools_scm]
+root = "../../.."
+tag_regex = "^firtool-(\\d+\\.\\d+\\.\\d+)?$"
```

Comment:
I'm not wild about tying these releases to firtool releases but they seem to be the only reasonable tags as of today.

Reply:
Yeah we can maybe do something different, but for now when a release with a firtool- tag is made, I'd like to generate these. We don't have to always do it this way though, and we don't have to keep baking the tag scheme into the project configuration... it can be supplied by the CI job through an environment variable or something.